src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone 2</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+
+<body>
+  <header>
+    <h1 class="logo">Movie <span class="slender">Somollier</span></h1>
+  </header>
```

Comment:
Please ensure that the `Home page header` and navigation are similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) according to the project requirement.

Reply:
The header has been updated